.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: node_js
 node_js:
 - 6.6.0
+sudo: false
 before_install:
-- npm install -g mocha
-- npm install -g gulp
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I’m a bit confused as to why we need this. Was under the impression that [Travis would use Yarn automatically](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Travis-CI-supports-yarn) if `package.json` and `yarn.lock` were found.